docs/developers.md

```diff
@@ -38,20 +38,21 @@ How to import a Gradle project:
 * [How to import Gradle project into IntelliJ IDEA](https://www.jetbrains.com/help/idea/gradle.html) (section "Importing a project
   from a Gradle model").
 
-**Note for Eclipse Users:**
+**Note for Eclipse users:**
 
-Eclipse needs to have Gradle build the generated source files before it can build the project.
-You can do this by running task `genBuildInfo` before importing the project.
+To successfully import the project in Eclipse, the complete project structure must be present,
+including Java source files built by Gradle.
+You can do this by running Gradle task `genBuildInfo` before importing the project.
 See [Building from sources](#building-from-sources) for how to run Gradle.
 
-To run the task within Eclipse after importing the logisim-evolution project:
+To run the task within Eclipse after importing the Logisim-evolution project:
 
-* Bring up the `Gradle Tasks` view, if it is not already showing, by selecting the Menu `Window/Show View/Other...`
+* Bring up the `Gradle Tasks` view, if it is not already showing, by selecting the menu `Window/Show View/Other...`
   and selecting `Gradle Tasks` under Gradle.
-* In the `Gradle Tasks` view, double-click on `logisim-evolution/build/genBuildInfo`.
+* In the `Gradle Tasks` view, double-click on `Logisim-evolution/build/genBuildInfo`.
```

Comment:
You checked it's spelled that way there?

Reply:
In Eclipse, it is spelled with a lowercase letter. Should I make that one lowercase?

Reply:
Yes, please make it exactly as it appears.

Reply:
Another thing, that we may use this opportunity to make one generic umbrella task (i.e. `genFiles`) that would in turn depend on all generators. Which would make things easier to manage in future.

Reply:
Done.